src/clincoded/static/components/variant_central/interpretation/segregation/evidenceTable.js

```diff
@@ -0,0 +1,478 @@
+            let deleteButton = this.getDeleteButton(row);
+            let buttons = <td key={`editDelete_${id}`} rowSpan={rowspan}>
+                {editButton} {deleteButton}
+            </td>
+            inner.push(buttons);
+        } else {
+            let emptyColumn = <td></td>
+            inner.push(emptyColumn);
+        }
+    },
+
+    /**
+     * Add the given criteria row data to table
+     *
+     * @param {string} criteria  The criteria name to be added
+     * @param {object} colNames  The table columns 
+     * @param {object} obj       The evidence source
+     * @param {number} key       Table column unique key
+     */
+    addAnotherCriteriaRow(criteria, colNames, obj, key) {
+        const emptyColumnNoTopBorder = <td style={{borderTop: 'none'}}></td>
+        let inner = [];
+        let outer = [];
+        let criteriaName = '';
+
+        colNames.forEach(col => {
+            let node = null;
+            let nodeContent = null;
+            let criteriaCodes = this.getCriteriaCodes(col);
+            if (col in obj) {
+                nodeContent = obj[col];
+            }
+            if (criteriaCodes.length > 0) {
+                // If this column is for given criteria, display its value
+                if (col === criteria) {
+                    node = <td key={`cell_${key++}`} style={{borderTop: 'none'}}>
+                        {nodeContent}
+                    </td>
+                    criteriaName = col;
+                }
+                else {
+                    // Other criteria, output blank column
+                    inner.push(emptyColumnNoTopBorder);
+                }
+            } else if (col === 'comments') {
+                // Display the comment for given criteria
+                let commentCol = criteriaName + '_comment';
+                nodeContent = null;
+                if (commentCol in obj) {
+                    nodeContent = obj[commentCol];
+                }
+                node = <td className='word-break-all' key={`cell_${key++}`} style={{borderTop: 'none'}}>
+                    {nodeContent}
+                </td>
+            }
+            // Add column
+            if (node) {
+                inner.push(node);
+            }
+        });
+        outer = <tr key={`row_${key++}`}>{inner}</tr>
+
+        return outer;
+    },
+
+    /**
+     * Display the evidences in the table for this panel
+     */
+    additionalEvidenceRows() {
+        let i = 0;  // Hack for unique key
+        let rows = [];
+
+        let colNames = this.state.tableFormat.cols.map(col => col.key);
+        // Don't read the kind_title property so we can handle each case separately.
+        colNames.splice(colNames.indexOf('_kind_title'), 1);
+        let tableData = this.props.tableData.filter(item => item.status != 'deleted');
+
+        tableData.forEach(row => {
+            if (this.showRow(row)) {
+                let obj = row.source.data;
```

Comment:
Can we rename this variable to something more descriptive than `obj`? Maybe `rowSourceData` or something like that